metal/end_model/em_defaults.py

```diff
@@ -5,7 +5,7 @@
     'show_plots': True,
 
     # Network
-    'batchnorm': True,
+    'batchnorm': False,
```

Comment:
Anything weird going on here :)?

Reply:
Haha, good check. This gets at a deeper question we should discuss sometime. For most applications (including our lstm_tests), with batchnorm on you do better; we want to make it "easy to do the right," which may mean leaving it on by default. But there's also something to be said for not adding in widgets that people don't know about, so default to no regularization, no dropout, no batchnorm, etc. until they explicitly add it, so that we know they're aware of it.

Reply:
I think as long as we document well what the defaults are, we're fine.  I'd argue for having it on but having a set of docs that explicate this very clearly.  We could even have a "raw" config and a "default" config, where the "raw" version has no bells/whistles and "default" has our opinion of best default practices.